nestkernel/nestmodule.cpp

```diff
@@ -1794,6 +1812,9 @@ NestModule::init( SLIInterpreter* i )
     "GetStructuralPlasticityStatus", &getstructuralplasticitystatus_function );
   i->createcommand( "Disconnect", &disconnect_i_i_lfunction );
   i->createcommand( "Disconnect_g_g_D_D", &disconnect_g_g_D_Dfunction );
+
+  i->createcommand( "SetStdpEps", &setstdpeps_dfunction );
```

Comment:
Should we have a `GetStdpEps` function, too?  Or maybe "hide" this in the kernel status dict after all?

Reply:
No, I don't think that we want a `GetStdpEps`. As this is a temporary fix, I would not like to make the variable visible in the kernel dict. I have added an info message to `SetStdpEps` though, which states the new value of `stdp_eps` and I have added two checks.